src/main/scala/org/broadinstitute/dsde/workbench/gpalloc/service/GPAllocService.scala

```diff
@@ -132,6 +133,15 @@ class GPAllocService(protected val dbRef: DbReference,
     cleanup
   }
 
+  private def overPetLimitOrProjectDeleted(project: String): Future[Boolean] = {
```

Comment:
I'd prefer to change `overPetLimit` to just list the SAs in the project and pull the logic that evaluates whether its over the limit up into this class, but I'm trying to minimize the amount of refactoring for this ticket

Reply:
Also considered renaming this to `shouldNukeProject`, but I think I prefer this naming since it describes why a project should get nuked